form/src/components/Execises.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+
+const Execises = () => {
```

Comment:
Since you're using react 18 version, there is no need to import react here. React does it for you. to improve the readability of the code, no need to use it.